<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Svetlana" failures="1" tests="4" name="Test" time="31.119" errors="0" timestamp="2025-05-19T17:53:09 ADT">
  <testcase classname="testCases.TC015_ProductCompareNumberDDT" name="verifyProductCompareNumberDDT" time="5.519"/>
  <testcase classname="testCases.TC015_ProductCompareNumberDDT" name="verifyProductCompareNumberDDT" time="5.439">
    <failure type="java.lang.AssertionError" message="Number of products not correct expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: Number of products not correct expected [1] but found [2]
at testCases.TC015_ProductCompareNumberDDT.verifyProductCompareNumberDDT(TC015_ProductCompareNumberDDT.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyProductCompareNumberDDT -->
  <testcase classname="testCases.TC015_ProductCompareNumberDDT" name="verifyProductCompareNumberDDT" time="5.425"/>
  <testcase classname="testCases.TC015_ProductCompareNumberDDT" name="verifyProductCompareNumberDDT" time="5.446"/>
</testsuite> <!-- Test -->
